aries_cloudagent/askar/profile.py

```diff
@@ -155,9 +155,11 @@ def __init__(
         """Create a new IndySdkProfileSession instance."""
         super().__init__(profile=profile, context=context, settings=settings)
         if is_txn:
-            self._opener = self.profile.store.transaction()
+            self._opener = \
+                self.profile.store.transaction(profile.context.settings.get("wallet.id"))
```

Comment:
`profile.context.settings.get("wallet.id")` Could be assigned to a variable outside if statements and used in both cases. 
I think It would be cleaner then and you could put the whole statement in one line

Reply:
Fixed